### PR TITLE
AGM-777 Big Sur 11.2 issues 

### PR DESCRIPTION
**Pull request by vbagirov**

### CODE REVIEW

File: 03.macos/05.solving-problems/04.big-sur-issues/docs.en.md [hunk 1]
```diff
@@ -6,26 +6,34 @@ taxonomy:
...
-* [Compatibility with Little Snitch 5](#little-snitch)
-* [Using "Automatic proxy" filtering mode](#automatic-proxy)
-* [Enabling Kernel Extension in Big Sur](#kernel-extension)
-* [Compatibility with Flutter app](#flutter)
+* [Currently existing problems (Big Sur 11.2)](#current)
+    * [Compatibility with local proxies](#local-proxies)
+        * [Configuring an upstream Shadowsocks proxy](#shadowsocks)
+        * [Cofiguring an upstream Surge proxy](#surge)
+    * [Compatibility with Cisco AnyConnect](#cisco)
+    * [Compatibility with Flutter app](#flutter)
```
Comment (ameshkov) [inline 03.macos/05.solving-problems/04.big-sur-issues/docs.en.md:14]: Flutter is not an app

Comment (vbagirov) [reply 03.macos/05.solving-problems/04.big-sur-issues/docs.en.md:14]: Software? Или просто Flutter и Flutter

File: 03.macos/05.solving-problems/04.big-sur-issues/docs.ru.md [hunk 2]
```diff
@@ -6,26 +6,33 @@ taxonomy:
...
+        * [Настройка вышестоящего прокси Shadowsocks](#shadowsocks)
+        * [Настройка вышестоящего прокси Surge](#surge)
+    * [Совместимость с Cisco AnyConnect](#cisco)
+    * [Совместимость с приложением Flutter](#flutter)
+    * [VPN-приложения с легаси API](#legacy-api)
+* [Уже исправленные проблемы](#fixed)
+    * [Совместимость с Little Snitch](#little-snitch)
+* [Альтернативы использованию Network Extension](#alternatives)   
+    * [Использование режима "Автоматический прокси"](#automatic-proxy)
+    * [Eереключение на Kernel Extension в Big Sur](#kernel-extension)
```
Comment (Chinaski1) [inline 03.macos/05.solving-problems/04.big-sur-issues/docs.ru.md:21]: Eереключение на Kernel Extension в Big Sur

File: 03.macos/05.solving-problems/04.big-sur-issues/docs.ru.md [hunk 3]
```diff
@@ -6,26 +6,33 @@ taxonomy:
...
```
Comment (ameshkov) [inline 03.macos/05.solving-problems/04.big-sur-issues/docs.ru.md:20]: Ошибка

File: 03.macos/05.solving-problems/04.big-sur-issues/docs.en.md (hunk 1 above)
Comment (vozersky) [reply 03.macos/05.solving-problems/04.big-sur-issues/docs.en.md:14]: просто Flutter

**Review by sfionov — APPROVED** [review]

**Review by vozersky — APPROVED** [review]

**Pull request merged**